appengine/go11x/static/main.go

```diff
@@ -20,6 +20,7 @@ var (
 	)
 )
 
+// [START main]
```

Comment:
The region tag should start with the product name and be in the samples tracker, which requires that. Otherwise, the sample will be untracked. If you let me know the updated name, I can update it here, if you prefer.